Tests/js/views/assets/ez-actionbarview-tests.js

```diff
@@ -37,49 +37,60 @@ YUI.add('ez-actionbarview-tests', function (Y) {
             this.view.destroy();
         },
 
-        "Should instantiate createContentActionView with contentType": function () {
-            var createContentActionFound = false;
+        _actionAssertions: function (testedActionId) {
```

Comment:
bad name, method should start with a verb, so here `_isActionCreated` or something like that.